packages/astro/src/assets/fonts/utils.ts

```diff
@@ -72,4 +72,31 @@ export function createCache(storage: Storage) {
 	};
 }
 
-export type CacheHandler = ReturnType<typeof createCache>;
\ No newline at end of file
+export type CacheHandler = ReturnType<typeof createCache>;
+
+/**
+ * The fonts data we receive contains urls or file paths we do no control.
+ * However, we will emit font files ourselves so we store the original value
+ * and replace it with a url we control
```

Comment:
Maybe an example can help this. I read it, and I'm not sure what's an URL that we can control.

Also, you mention the URL but the return type is a string. Which one is correct? If it's really an URL, shouldn't we use the `URL` type?

Can we also document what the collect function is supposed to do?

Reply:
I improved the comment